conftest.py

```diff
@@ -284,9 +105,11 @@ def pcie_devices(request, device_params):
     import tt_lib as ttl
 
     num_devices = ttl.device.GetNumPCIeDevices()
+    device_ids = [i for i in range(num_devices)]
+    request.node.pci_ids = [ttl.device.GetPCIeDeviceID(i) for i in device_ids]
```

Comment:
This fixture is technically for pcie devices so we shouldn't need to call `GetPCIeDeviceID` on the device ids, unless the fixture itself is busted and grabbing non-mmio devices.

Reply:
Ahh, makes sense. Changed, please review 🙏